docs/application/native/guides/app-management/cion.md

```diff
@@ -0,0 +1,235 @@
+# Cion
+
+The Cion API provides functionality to communicate with the other devices.
+
+The main features of the Cion API include:
+
+- Communicating with other applications in server-client style.
+
+  You can communicate with other applications in [server-client](#server_client) style.
+
+- Communicating with other applications in group style.
+
+  You can communicate with other applications in [group](#group) style.
+
+- Supporting TIDL
+
+  You can generate communication code using [TIDL](#tidl).
+
+## Prerequisites
+
+1.  To use the Cion API (in [mobile](../../api/mobile/latest/group__CAPI__CION__MODULE.html), [wearable](../../api/wearable/latest/group__CAPI__CION__MODULE.html) and [iot](../../api/iot-headed/latest/group__CAPI__CION__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
```

Comment:
In case of native and web API, they use their own LNB style. Also the stg seems not be updated yet to apply "latest".

I think you can see what is the LNB style of Native API on public.
 - https://docs.tizen.org/application/native/api/mobile/latest/group__CAPI__CION__MODULE.html
 - https://docs.tizen.org/application/native/api/wearable/latest/group__CAPI__CION__MODULE.html
 - https://docs.tizen.org/application/native/api/iot-headed/latest/group__CAPI__CION__MODULE.html